proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -240,6 +247,11 @@ message UnbondingDelegationEntry {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
     (gogoproto.nullable)   = false
   ];
+  // Incrementing id that uniquely identifies this entry
+  uint64 unbonding_id = 5;
```

Comment:
I agree. We just wanted to be consistent with the validator's field. All three are unbonding operations that have an ID. We refer to that ID as the unbonding ID, regardless of the type of operation (i.e., undelegation, redelegation, or validator unbonding).

Reply:
So can we get rid of the prefix?

Reply:
I'd prefer to keep it, however, it's not a strong preference. :)

Reply:
I think we should keep it for the reason Marius said. It's known as unbonding_id wherever it is used

Reply:
My argument is the type implies it, e.g. `unbonding.ID`. Just like how would you wouldn't have a `NewFoo` constructor in a `foo` package as `foo.New()` implies it.

But it's all good, not a hill im gonna die on. Feel free to resolve.

Reply:
Well, it's also used on redelegations and validator unbondings, but it has the same meaning and use everywhere

Reply:
Not sure I follow, but argument I'm making is there is no need for the prefix since the type implies it. Again, feel free to resolve and close the convo 👍